ccp/modules/dnpm-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ -n "${ENABLE_DNPM}" ]; then
+	log INFO "DNPM setup detected (Beam.Connect) -- will start Beam.Connect for DNPM."
```

Comment:
```suggestion
	log INFO "DNPM setup detected (Beam.Connect) -- will start Beam and Beam.Connect for DNPM."
```

Reply:
Here (in ccp), Beam is already running and only Beam.Connect is started additionaly for DNPM